<launch>
        <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="world_name" value="$(find control)/worlds/empty.world"/>
        </include>
        <include file="$(find control)/launch/spawn_quadrotor.launch" >
        <arg name="model" value="$(find cvg_sim_gazebo)/urdf/quadrotor_sensors.urdf.xacro"/>
        </include>
        <include file="$(find control)/launch/orb_slam2_mono_sim.launch" >
        </include>
        <!-- <include file="$(find ardrone_joystick)/launch/teleop.launch" >
        </include> -->
	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find control)/rviz/config_file.rviz" />
        <!-- <include file="$(find control)/launch/octomap_mapping.launch" >
        </include> -->


        <!-- <node pkg="nodelet" type="nodelet" name="pcl_manager" args="manager" output="screen" />
        <node pkg="nodelet" type="nodelet" name="statisticalOutliers" args="load pcl/StatisticalOutlierRemoval pcl_manager" output="screen">
                <remap from="~input" to="/perception/object_cluster" />
                <rosparam>
                 mean_k: 50
                 stddev: 0.5
                 negative: False
                </rosparam>
        </node> -->
        <!-- <node pkg="nodelet" type="nodelet" name="passthrough" args="load pcl/PassThrough pcl_manager" output="screen">
        <remap from="~input" to="/statisticalOutliers/output" />
                <rosparam>
                filter_field_name: x
                filter_limit_min: -0.5
                filter_limit_max: 0.5
                filter_limit_negative: False
                </rosparam>
        </node> -->
</launch>
